src/utils/Utils.ts

```diff
@@ -36,3 +36,20 @@ export function toLowerCasePropertyNames(input: any): any {
     return result;
   }, {});
 }
+
+/**
+ * Get UTC offset in (+|-)HH:mm format
+ * E.g. For Asia/Calcutta timezone, +05:30 UTC offset value is returned
+ */
+export function getUTCOffsetString(utcOffsetInteger: number): string {
```

Comment:
- Can function be named to `getFormattedOffset(utcOffset: number): string`?
- Do we have to consider DST difference?

Reply:
getTimezoneOffset considers daylight savings as well. Ref [this](https://stackoverflow.com/questions/11887934/how-to-check-if-dst-daylight-saving-time-is-in-effect-and-if-so-the-offset).

Reply:
Will rename to getFormattedOffset